tests/conftest.py

```diff
@@ -8,6 +8,7 @@
 import pytest
 from multidict import CIMultiDict
 
+from lbz import jwt_utils
```

Comment:
Wait a moment, you imported the whole `jwt_utils` module and additionally `encode_jwt()` below? Why?